### DISCUSSION TITLE
关于全局设置中本地DNS劫持的问题

### DISCUSSION THREAD

**Discussion by eightsheep67** [vernesong/OpenClash#2541] [General]
这个本地DNS劫持设置，如果打开的，会导致我XBOX联机测试数据包丢失率为100%
我尝试设置各种规则，如关键字xboxlive、microsoft走直连，XBOX的IP走直连，规则生效，但数据包丢失率仍为100%
然后我关闭了本地DNS劫持，不管是直连，还是走代理，丢包率都正常了。
但是路由下载规则集的时候显示下载失败，用openwrt网络诊断发现本机无法走代理，那肯定会影响插件的更新功能。

怎么办？？？

**Comment by eightsheep67** [comment]
最后添加了DNS高级设置解决。。。
![屏幕截图 2022-07-10 162100](https://user-images.githubusercontent.com/70835718/178137100-f3cba040-78b2-4261-8bca-d0b8ac39b156.png)

**Reply by kiligone** [reply]
你是开udp转发了吗，我是开了udp转发，xbox测速丢包率就100%

**Reply by eightsheep67** [reply]
没开，UDP转发也不能开。

**Reply by a137594942** [reply]
按照你这样填写 xbox 测试联机能通过 但是光环联机 无法连接

**Comment by eightsheep67** [comment]
用redir-host，不要用fakeip



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: a137594942 ***@***.***&gt;
发送时间: 2022年10月11日 20:06
收件人: vernesong/OpenClash ***@***.***&gt;
抄送: eightsheep ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [vernesong/OpenClash] 关于全局设置中本地DNS劫持的问题 (Discussion #2541)





 
按照你这样填写 xbox 测试联机能通过 但是光环联机 无法连接
